packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -170,7 +224,13 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "closeButtonTooltip": r"Fermer",
     "nextMonthTooltip": r"Mois Suivant",
     "previousMonthTooltip": r"Mois précédent",
+    "nextPageTooltip": r"Page suivante",
+    "previousPageTooltip": r"Page précédente",
     "licensesPageTitle": r"Licences",
+    "pageRowsInfoTitle": r"$firstRow–$lastRow de $rowCount",
+    "pageRowsInfoTitleApproximate": r"$firstRow–$lastRow d'environ $rowCount",
+    "rowsPerPageTitle": r"Lignes par page:",
+    "selectedRowCountTitleOther": r"$selectedRowCount articles sélectionnés",
```

Comment:
numbers in french have to be localized, int.toString() doesn't output valid numbers for french.

Reply:
I'm using NumberFormat from the intl package now.

Reply:
I'm using NumberFormat from the intl package now.